build.gradle.kts

```diff
@@ -7,6 +7,9 @@ subprojects {
   apply(plugin = "java")
   apply(plugin = "com.diffplug.spotless")
 
+  group = "org.stellar"
+  version = "1.0.1"
```

Comment:
Are we sure we want to publish under v1? That’s communicating that we won’t break the interface in any other v1 release. Maybe we use v0.1? And release 1.0 when we know our interface is solid. Up to you though.

Reply:
Just renamed the version as we speak. The version is changed to 0.1.0